src/v0/shift_plan.ts

```diff
@@ -62,22 +62,22 @@ export class ShiftPlan extends ThBaseHandler {
 
       return {
         msg: response.data.msg,
-        data: response.data.results[0],
+        data: response.data.results,
         metadata: { count: response.data.count }
       }
     } catch (error: any) {
       throw new ShiftPlanFetchFailed(error.message, { error })
     }
   }
 
-  async put (branchId: string, shiftPlan: ShiftPlanObject): Promise<ShiftPlanResponse> {
+  async put (branchId: string, shiftPlanOptions: ShiftPlanUpdateOptions): Promise<ShiftPlanResponse> {
```

Comment:
Any preferences on how the `shiftPlanOptions` argument and its type are named? I see the pattern is to use main entity ( eg PriceBook or Product ) but in this case the payload that put request accepts is not the same as the one returned from GET

Reply:
`ShiftPlanUpdateOptions` is fine